harness/determined/horovod.py

```diff
@@ -143,6 +143,8 @@ def create_run_command(
         create_hostlist_arg(num_gpus_per_machine, ip_addresses),
         "--start-timeout",
         str(constants.HOROVOD_STARTUP_TIMEOUT_SECONDS),
+        "--gloo-timeout-seconds",
+        str(constants.HOROVOD_GLOO_TIMEOUT_SECONDS),
```

Comment:
idea: wonder if we should actually do some scaling by the number of instances..

Reply:
IMO that would be unnecessary complexity. There really isn't any downside to having a 4-minute timeout (famous last words) for this.

Reply:
I was more thinking of if we go even more instances, are we going to have to keep bumping this. But I'm fine either way

Reply:
That's a good question. 4-minutes should already give a lot of space to scale up so I'm inclined to handle that case if/when we hit in the future.